### PR TITLE
Integrating numeric data

### PR DESCRIPTION
**Pull request by vcaf**

### CODE REVIEW

**Comment by vcaf** [comment]
Note: Because I have added regex to remove all unwanted none integers, it also removes edge cases as in 25 to 30 people that converts the output to 2530.

File: Prototype/Frontend/src/pages/form/components/FormField.tsx [hunk 1]
```diff
@@ -25,6 +26,9 @@ const FormField = ({formItem, answer, showConfidence} : FormFieldProps) : JSX.El
     case FieldType.TEXT:
       field = <TextField formItem={formItem} answer={answer} showConfidence={showConfidence} />;
       break;
+      case FieldType.NUMERIC:
```
Comment (vcaf) [inline Prototype/Frontend/src/pages/form/components/FormField.tsx:29]: Indentation that has to be fixed

**Comment by Eliturbo** [comment]
> Note: Because I have added regex to remove all unwanted none integers, it also removes edge cases as in 25 to 30 people that converts the output to 2530.

Is there any way we can fix this? Maybe we can only take the first number if we see multiple in a sentence divided by another string

**Comment by Eliturbo** [comment]
Other than that, LGTM!

**Comment by vcaf** [comment]
Added that it only takes the first digit: 

Screenshot (1618)

**Review by Eliturbo — APPROVED** [review]

**Pull request merged**